conf/airframes/examples/quadrotor_lisa_m_2_pwm_spektrum.xml

```diff
@@ -57,24 +57,16 @@
   </commands>
 
   <section name="MIXING" prefix="MOTOR_MIXING_">
-    <define name="TRIM_ROLL" value="0"/>
```

Comment:
Sure it is, but the default for the trim is zero, so you might as well omit it...